src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,64 @@
+package view;
+
+import static view.Command.END;
+import static view.Command.MOVE;
+import static view.Command.START;
+
+import domain.board.ChessBoard;
+import domain.piece.Piece;
+import domain.piece.Type;
+import domain.position.File;
+import domain.position.Position;
+import domain.position.Rank;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class OutputView {
+    private static final Map<Type, String> PIECE_DISPLAY = Map.of(
+            Type.PAWN, "p",
+            Type.KNIGHT, "n",
+            Type.BISHOP, "b",
+            Type.ROOK, "r",
+            Type.QUEEN, "q",
+            Type.KING, "k"
+    );
+
+    public void printStartingMessage() {
+        System.out.println("> 체스 게임을 시작합니다.");
+        System.out.printf("> 게임 시작 : %s%n", START.getName());
+        System.out.printf("> 게임 종료 : %s%n", END.getName());
+        System.out.printf("> 게임 이동 : %s source위치 target위치 - 예. %s b2 b3%n", MOVE.getName(), MOVE.getName());
+    }
+
+    public void printBoard(ChessBoard chessBoard) {
+        Map<Position, Piece> board = chessBoard.getBoard();
+        for (int rank = 8; rank >= 1; rank--) {
+            printBoardRow(board, Rank.fromNumber(rank));
+        }
+    }
+
+    private void printBoardRow(Map<Position, Piece> board, Rank targetRank) {
+        List<String> strings = new ArrayList<>(Collections.nCopies(8, "."));
+        for (var positionAndPiece : board.entrySet()) {
+            Position position = positionAndPiece.getKey();
+            Piece piece = positionAndPiece.getValue();
+            Rank rank = position.rank();
+            File file = position.file();
+
+            if (rank == targetRank) {
+                strings.set(file.order(), pieceDisplay(piece));
+            }
+        }
+        System.out.println(String.join("", strings));
+    }
+
+    private String pieceDisplay(Piece piece) {
+        String pieceName = PIECE_DISPLAY.get(piece.getType());
+        if (piece.getColor().isBlack()) {
```

Comment:
```suggestion
        if (piece.isBlack()) {
```
이런식으로 color 를 꺼내지 않고 piece 가 직접 확인하도록 하는건 어떨까요?

Reply:
`piece`에게 물어보고 싶었어요! 하지만 깔끔하게 추상화한 `piece`에 유틸리티성 메서드를 넣고 싶지 않았어요. 또한 `Color.class`는 `public`으로 코드 전반에 퍼져 있는 VO이기 때문에 직접 꺼냈어요. 

위 근거로 작성한 코드인데, 피케이의 의견이 궁금해요! `Piece`에게 물어보는 것이 더 좋은 코드일까요?

Reply:
이왕이면 getter 는 데이터를 출력할 때만 사용하고, 그 외의 상황에서는 사용하는 객체에게 직접 그 정보를 알려달라고 하는게 좋다고 생각해요. '더 좋은 코드' 인지는 사람마다 생각하는게 다 다를 것 같은데 만약 망쵸가 생각했을 때 `getColor()` 로 꺼내오는게 유지보수 할 때 더 좋다고 생각한다면 그 방식으로 진행하셔도 괜찮습니다!

그런데 망쵸의 `깔끔하게 추상화한 piece 에 유틸리티성 메서드를 넣고 싶지 않았다` 는게 어떤 의미인지 궁금해요.
깔끔하게 추상화한 형태는 무엇이고, 유틸리티성 메서드는 무엇인가요?

Reply:
다음은 모든 피스를 추상화한 `piece` 인터페이스 코드에요. 

```java
public interface Piece {
    void validateMovement(Position source, Position target, Piece other);
    Color color();
    Type type();
}
```

피케이가 말씀해주신 대로 객체에게 질문하도록 코드를 수정하면 `piece.getColor().isBlack()` -> `piece.isBlack()` 이런 식으로 변경될 것 같아요. 그럼 Piece 인터페이스 내부에 `isBlack()`이라는 메서드가 추가될 거에요. 

`isBlack()`는 뷰에서 사용하는 메서드로 핵심 로직과는 무관해요. 또한 `Piece` 내부에 `Color`를 꺼내는 메서드가 존재하기 때문에 없어도 구현하는 데에 지장이 없어요. 

추상화하게 되면 모든 하위 계층에서 해당 메서드를 구현해야 해요. 이 비용보다 이미 존재하는 `Color`를 꺼내는 메서드를 사용하는 편이 낫다고 생각했어요. 

> 깔끔하게 추상화한 형태는 무엇이고, 

Piece 인터페이스를 두고 남긴 코멘트였어요! _깔끔하다_ 는 제 기준에서 나름 열심히 추상화했고 그 뿌듯함이 담긴  표현이에요. 

> 유틸리티성 메서드는 무엇인가요? 

위에서 언급한 `isBlack()`과 같이 핵심 로직과 무관하게 편의를 위해 존재하는 메서드를 두고 남긴 코멘트였습니다.

Reply:
아 제가 쟐못 본 부분이 있었어요. isBlack 이 로직에서도 사용되는 줄 알았는데 출력시에만 사용되는군요. 위 리뷰에서 남긴 것처럼 저도 출력할 때 getter 쓰는건 찬성입니다! 잘못봐서 죄송해요 🙇‍♂️

별개로 본인이 작성하는 코드에 대해 깊은 고민을 하고 만족스러운 형태로 만들어내는 모습이 정말 좋네요 🙂